lib/streamlit/components/v1/components.py

```diff
@@ -327,7 +327,7 @@ def get(self, path: str) -> None:
             with open(abspath, "r", encoding="utf-8") as file:
                 contents = file.read()
         except (OSError, UnicodeDecodeError) as e:
-            self.write(f"{path} read error: {e}")
+            self.write(f"read error: {e}")
```

Comment:
To be extra safe, I think we should also remove the exception from the interpolated string here. (So just: `self.write("read error")`). We don't need to leak details to the browser about what's going on under the hood. (We could log the exception to the console, for the site admin to see.)